packages/touchpoint-ui/src/bidirectional/commandHandler.ts

```diff
@@ -101,6 +101,10 @@ export const commandHandler = (
           )!;
           handler(event.payload);
         }
+        // eslint-disable-next-line no-console
+        console.warn(
+          `No command handler was defined for the ${event.action} action.`,
```

Comment:
```suggestion
        debug(
          `No custom command handler was defined for the %o action.\n\n%cTip: Set up a handler with \nsetCustomBidirectionalCommands([{ action: "${event.action}", handler() { }}])`, event.action, "font-style: italic; font-size: 90%"
```

<img width="616" height="94" alt="Screenshot 2025-09-30 at 11 25 15" src="https://github.com/user-attachments/assets/18c9c187-c65f-4fa5-995e-5225a0ed62fb" />

Something like that?